google-cloud-bigquery/acceptance/bigquery/dataset_test.rb

```diff
@@ -292,6 +296,33 @@
     table_avro.encryption.must_equal encrypt_config
   end
 
+  it "imports data from a local Parquet file and creates a new table without a schema with load" do
+    result = dataset.load table_parquet_id, local_parquet_file
+    result.must_equal true
+  end
+
+  it "imports data from GCS Parquet file and creates a new table with load" do
+    result = dataset.load(
+        table_parquet_id,
+        "gs://cloud-samples-data/bigquery/us-states/us-states.parquet")
+    result.must_equal true
+  end
+
+  it "imports data from GCS Parquet file and creates a new table with encryption with load" do
```

Comment:
You can probably drop this test if you want. The fact that the other encryption test is Avro is more a side-effect that Avro is both fast to load and has an embedded schema.

Reply:
ok, will do.